docker-compose.yml

```diff
@@ -26,10 +26,11 @@ services:
     memswap_limit: 512m
     privileged: false
     ulimits:
-      nproc: 10
+      # nproc: 10
```

Comment:
Is this causing issues? I added as part of one of the security requirements to limit resources

Reply:
Yes, when I was trying to run docker-compose locally, i was getting "resource unavailable" errors that were only alleviated by removing this limit. Where did you find the requirement for setting the value to 10? I cannot find a recommendation for a specific value.

Reply:
I don't think it has to be 10, it should just have a max to prevent a bunch of child processes from being spawned (I think it's unlimited at the moment for linux systems)